VirtoCommerce.Platform.Data.Security/Converters/ApplicationUserConverter.cs

```diff
@@ -18,7 +18,15 @@ public static ApplicationUserExtended ToCoreModel(this ApplicationUser applicati
             retVal.InjectFrom(applicationUser);
             retVal.InjectFrom(dbEntity);
             retVal.UserState = EnumUtility.SafeParse(dbEntity.AccountState, AccountState.Approved);
-
+            if (applicationUser.Logins != null)
+            {
+                retVal.Logins = new ApplicationUserLogin[10];
```

Comment:
10 - is a magical number? Why not 666? (sarcasm) Just remove this line

Reply:
its my lucky number